Functions/xASL_qc_CollectQC_ASL.m

```diff
@@ -52,7 +52,11 @@
     % The determinant of the current matrix and old matrix should be the same,
     % otherwise this is suspicious of a left-right flip.
     PathOrientationResults = fullfile(x.SESSIONDIR,'xASL_qc_PrintOrientation_RigidRegASL.tsv');
-    ASL = xASL_im_DetermineFlip(x, iSubject, PathOrientationResults, ASL);
+    ASL.LR_flip_YesNo = uint8(xASL_im_DetermineFlip(PathOrientationResults));
```

Comment:
We could add this parameter to the new sub-structs of x while we're at it. So Why not use `x.modules.asl.LR_flip_YesNo` instead 😃

Reply:
Ah sorry, I saw that this ASL struct is stored in x.Output later on. Then my request doesn't really make sense 👍 I'll mark it as resolved.

Reply:
Yes the `x.output` carries all QC stuff, we could resort this later.